cmd/influxd/run/server.go

```diff
@@ -136,21 +132,23 @@ func NewServer(c *Config, buildInfo *BuildInfo) (*Server, error) {
 	// 0.11 we no longer use peers.json.  Remove the file if we have one on disk.
 	os.RemoveAll(filepath.Join(c.Meta.Dir, "peers.json"))
 
-	// load the node information
-	metaAddresses := []string{nodeAddr}
-	if !c.Meta.Enabled {
-		metaAddresses = c.Meta.JoinPeers
-	}
-
-	node, err := influxdb.LoadNode(c.Meta.Dir, metaAddresses)
+	node, err := influxdb.LoadNode(c.Meta.Dir)
 	if err != nil {
 		if !os.IsNotExist(err) {
 			return nil, err
 		} else {
-			node = influxdb.NewNode(c.Meta.Dir, metaAddresses)
+			node = influxdb.NewNode(c.Meta.Dir)
```

Comment:
Feels like there is some future refactoring here yet.  I'm not sure what purpose a node without a valid ID really serves.  Just thinking out loud on this one.  No action needed at this time.